### ISSUE TITLE
DAILY_KEYS_MINI_GAME_WRONG. Dont use it now. U may get ban!!

### ISSUE THREAD

**Issue by Ashwin3454** [shamhi/HamsterKombatBot#2325]
{"error_code":"DAILY_KEYS_MINI_GAME_WRONG","error_message":"DailyKeysMiniGame is wrong NjkxMTAyMzE1OHw2ODgzMTA0MjYwfFRpbGVzfDM0NTAxMDEwMjh8RnlWM09qOFFpbENlSHc3S3RPaE1IRTJhaU1WWlo3blhxMmVjY3o4SjNuZz0="}

(The api has changed. Dont use it now. U may get ban!!)

**Comment by semeer37** [comment]
Is this true?

**Comment by HollySheeeet** [comment]
> Is this true?

No, but you can still get banned for using the software, which is why he said 'maybe.' (But maybe not..)
